docs/configuration.md

```diff
@@ -13,9 +13,9 @@ Keep in mind, that each option used on the command line will **completely replac
 ## Usage of Globbing Expressions on options
```

Comment:
I see you've formatted this file with something. Please undo that, as this will pollute our history.

Reply:
not sure why the file was exactly formatted, but it should be fine now

Reply:
Thanks! It's not your fault though, it is normal to use a formatted in your editor. We should use formatter to verify it in the pipeline...